File: abc140_c.cpp
// https://atcoder.jp/contests/abc140/tasks/abc140_c
#include<iostream>
#include<algorithm>
//#include <numeric>
#include<vector>
//#include<map>
//#include<tuple>
//#include<set>
//#include<queue>
//#include<regex>
//#include <bitset>

// #include <bits/stdc++.h>
#define rep(i,n) for (int i = 0; i < (n); ++i)
using namespace std;
typedef long long ll;

int main() {
	int N;
	cin >> N;

	vector<int> B(N-1);
	rep(i, N-1) {
		cin >> B[i];
	}

	int ans = B[0] + B[N-2];
	rep(i, N-2) {
		ans += min(B[i], B[i+1]);
	}

	cout << ans << endl;
	return 0;
}

File: atcoder/abc140_c.py
# https://atcoder.jp/contests/abc140/tasks/abc140_c

N = int(input())
B = list(map(int, input().split()))

ans = B[0] + B[-1]
for i in range(N-2):
    ans += min(B[i], B[i+1])

print(ans)
